### DISCUSSION TITLE
Support Tiktok as an Auth Provider

### DISCUSSION THREAD

**Discussion by sambowenhughes** [supabase/supabase#10846] [Feature Requests]
Is there anything on the backlog for adding Tiktok as a verified auth provider?

**Comment by ghost** [comment]
im still amazed this is not an available feature yet.

**Reply by michaeltomasik** [reply]
They made 2 PRs for it 
https://github.com/supabase/auth/pull/269 Although this one got closed, someone made a new one only with tiktok here
https://github.com/supabase/auth/pull/1807
